buildConfig.tsconfig.json

```
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "jsx": "react",
+    "target": "ES2016",
+    "module": "commonjs",
+    "lib": ["ESNext", "dom"],
+    "moduleResolution": "node",
+    "esModuleInterop": true,
+    "strict": true,
+    "noImplicitAny": true,
+    "skipLibCheck": true,
```

@rezrah I had to add `"skipLibCheck": true,` to avoid an error:

> node_modules/vite/dist/node/index.d.ts:5:41 - error TS2307: Cannot find module 'rollup/parseAst' or its corresponding type declarations.
  There are types at '/Users/lukasoppermann/GitHub/primitives/node_modules/rollup/dist/parseAst.d.ts', but this result could not be resolved under your current 'moduleResolution' setting. Consider updating to 'node16', 'nodenext', or 'bundler'.
> 
> 5 export { parseAst, parseAstAsync } from 'rollup/parseAst';
>                                           ~~~~~~~~~~~~~~~~~
> Found 1 error in node_modules/vite/dist/node/index.d.ts:5

However, I read that this is more like a "hack". What do you think about this?

---

🤔 yeah, disabling that is quite problematic. To unblock you, maybe disable for now, but add a comment above that line along with a TODO to make sure we follow up and fix it. Changing to `bundler` is a common way to fix it, but i'm not sure what rollup is used for unless I dig into the relevant code.

---

Comment could be something like this:

```suggestion
    // FIXME: Disabled in https://github.com/primer/primitives/pull/1032
    "skipLibCheck": true,
```

---

Changing to `bundler` sadly does not solve it. 

I like the comment idea.